src/pages/signin/SignInPageLayout/Footer.js

```diff
@@ -160,8 +160,8 @@ function Footer(props) {
 
     return (
         <View style={[styles.flex1]}>
-            <View style={[props.isSmallScreenWidth ? StyleUtils.getBackgroundColorStyle(themeColors.signInPage) : {}]}>
-                {props.isSmallScreenWidth ? (
+            <View style={[...(isVertical ? [StyleUtils.getBackgroundColorStyle(themeColors.signInPage), styles.overflowHidden] : [{}])]}>
```

Comment:
i think we can move this our from render and add const footerWrapper = isvertical ? StyleUtils.getBackgroundColorStyle(themeColors.signInPage), styles.overflowHidden] : null
and use <View style={footerWrapper}
something like that